scripts/setup.sh

```diff
@@ -53,7 +53,7 @@ if [ ${PWD##*/} != $EXPECTED_PWD ]; then
 fi
 
 export OPPIA_DIR=`pwd`
-export COMMON_DIR=$OPPIA_DIR/..
+export COMMON_DIR=$(cd $OPPIA_DIR/..; pwd)
```

Comment:
I added a comment to clarify; per our offline discussion, the command is needed.